frontend-student/src/components/home/Home.jsx

```
@@ -1,49 +1,327 @@
+          >
+            Transform education through our innovative platform that bridges
+            students, educators, and content in a seamless digital ecosystem
+          </motion.p>
+          <motion.div
+            className='flex gap-4 mt-6'
+            initial={{ opacity: 0, y: 20 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={{ duration: 0.8, delay: 0.9 }}
+          >
+            <button className='px-6 py-3 bg-blue-600 text-white rounded-lg hover:bg-blue-700 transition'>
+              Get Started
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)